apps/docs/versioned_docs/version-3.1.1/page-builder.md

```
@@ -0,0 +1,247 @@
+# Page Builder
+
+The page builder enables content editors to compose pages from reusable components in Strapi, which are automatically rendered by the Next.js frontend.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)